pkg/httputil/client_test.go

```diff
@@ -106,14 +112,12 @@ func TestNewRateLimitedClient(t *testing.T) {
 		}
 
 		if tt.err == "" {
-			took := time.Since(now)
+			took := time.Since(start).Round(time.Second)
 			expectedTook := time.Duration(0)
```

Comment:
I know you didn't name these, but since we're in cleanup mode, can you rename these variables to `observedDuration` and `expectedDuration`?

Reply:
Make sense